src/states/node.rs

```diff
@@ -1285,15 +1192,16 @@ impl Node {
         use crate::messages::MessageContent::*;
         use crate::Authority::{Client, ManagedNode, PrefixSection, Section};
 
-        if !self.chain.is_member() && !self.is_first_node {
+        if !self.chain.is_member() {
```

Comment:
Why did this change? Is it because of https://github.com/maidsafe/routing/pull/1613/commits/55136aa72e9e82dc298b5021fda7170ed7846b68 ?

Could you say a bit more about this change in the PR description, I'm not sure I see all the implication, because is_first_node is still a member, so some thing still needs it.

Reply:
> Is it because of 55136aa

Precisely. First node now has its chain initialised from the beginning, so `chain.is_member()` does always return `true` for it.